src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +39,9 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
 
     @Override
-    Player getUser();
-
+    User getUser();
```

Comment:
There is no reason to _not_ return player. It gives you more specifics and it is overall more useful. You are already changing getEntity, so you need to change getUser as well, if for nothing else other than consistency.